packages/react-components/react-rating-preview/src/components/RatingItem/useRatingItem.tsx

```diff
@@ -0,0 +1,132 @@
+import * as React from 'react';
+import { getIntrinsicElementProps, slot, useMergedRefs } from '@fluentui/react-utilities';
+import { useFocusWithin } from '@fluentui/react-tabster';
+import type { RatingItemProps, RatingItemState } from './RatingItem.types';
+import { useRatingContextValue_unstable } from '../../contexts/RatingContext';
+
+/**
+ * Create the state required to render RatingItem.
+ *
+ * The returned state can be modified with hooks such as useRatingItemStyles_unstable,
+ * before being passed to renderRatingItem_unstable.
+ *
+ * @param props - props from this instance of RatingItem
+ * @param ref - reference to root HTMLElement of RatingItem
+ */
+export const useRatingItem_unstable = (props: RatingItemProps, ref: React.Ref<HTMLSpanElement>): RatingItemState => {
+  const context = useRatingContextValue_unstable();
```

Comment:
If you change the default value of RatingContext to `undefined` then I think you can just change this to
```suggestion
  const context = useRatingContextValue_unstable() || {};
```

If TypeScript doesn't like that, then you can leave this line as it is, and change the places you're using it to use `?.`, like: `context?.value`